### PR TITLE
Upgrade eslint prettier

### PR DESCRIPTION
**Pull request by felixsaintgenis**
list of desactivated rules : 
```
    "react/require-default-props": 0,
    "react/destructuring-assignment": 0,
    "react/default-props-match-prop-types": 0,
    "jsx-a11y/click-events-have-key-events": 0,
    "prefer-destructuring": 0,
    "object-curly-newline": 0,
    "implicit-arrow-linebreak": 0,
    "sx-curly-brace-presence": 0,
    "operator-linebreak": 0,
    "jsx-a11y/label-has-associated-control": 0, // input must be nested in the label and htlmfor identified.
    "jsx-a11y/label-has-for": 0, // input must be nested in the label and htlmfor identified
```

This PR is open to discussion and we should decide together what desactivated rule must be reactivated in the project.

### CODE REVIEW

File: assembl/static2/.eslintrc.json [hunk 1]
```diff
@@ -6,29 +6,62 @@
...
+      "error",
+      {
+        "devDependencies": [
+          "**/stories/**/*.jsx",
+          "**/tests/**/*.*",
+          "**/__test__/*"
+        ]
+      }
+    ],
+    "react/require-default-props": 0,
```
Comment (pascalhuynh) [inline assembl/static2/.eslintrc.json]: could you reorder the rules by ascending alphabetical order ?

File: assembl/static2/js/app/actions/adminActions/landingPage.js [hunk 2]
```diff
@@ -25,13 +25,7 @@ export const moveLandingPageModuleDown: moveLandingPageModuleDownAction = id =>
...
 });
 
-export const createLandingPageModule = (
-  id: string,
-  identifier: string,
-  numberOfDuplicatesModules: number,
-  title: string,
-  order: number
-): actionTypes.createLandingPageModule => ({
+export const createLandingPageModule = (id: string, identifier: string, numberOfDuplicatesModules: number, title: string, order: number): actionTypes.createLandingPageModule => ({
```
Comment (pascalhuynh) [inline assembl/static2/js/app/actions/adminActions/landingPage.js]: is it normal that parameters are all set inline now ?

File: assembl/static2/js/app/components/administration/discussion/preferences/index.jsx [hunk 3]
```diff
@@ -8,10 +8,10 @@ import { Field, FormSpy } from 'react-final-form';
 import classnames from 'classnames';
 import Loader from '../../../common/loader';
 import LoadSaveReinitializeForm from '../../../form/LoadSaveReinitializeForm';
-import AdminForm from '../../../../components/form/adminForm';
-import CheckboxFieldAdapter from '../../../../components/form/checkboxFieldAdapter';
+import AdminForm from '../../../form/adminForm';
```
Comment (pascalhuynh) [inline assembl/static2/js/app/components/administration/discussion/preferences/index.jsx]: were the components moved ?

File: assembl/static2/js/app/components/administration/discussion/timelineConfiguration/index.jsx [hunk 4]
```diff
@@ -89,7 +84,10 @@ class TimelineFields extends React.Component<Props> {
                         const title = fieldValue && fieldValue.title ? fieldValue.title[editLocale] || '' : '';
                         return (
                           <span>
-                            <Translate value={titleMsgId} count={idx + 1} /> &mdash; {title}
+                            <Translate value={titleMsgId} count={idx + 1} />
+                            {' '}
```
Comment (pascalhuynh) [inline assembl/static2/js/app/components/administration/discussion/timelineConfiguration/index.jsx]: why not use interpolation ?

File: assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx [hunk 5]
```diff
@@ -198,29 +191,21 @@ const DumbVoteProposalForm = ({
...
                 ) : (
                   <Translate value="administration.voteProposals.gauge" number={number} />
-                )}{' '}
-                -{' '}
-                {getEntryValueForLocale(
-                  voteSpecTemplate.get('instructionsEntries'),
-                  editLocale,
-                  getEntryValueForLocale(voteSpecTemplate.get('instructionsEntries'), 'en')
                 )}
+                {' '}
```
Comment (pascalhuynh) [inline assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx]: why not use interpolation ? or a constant ?

File: assembl/static2/js/app/components/common/footer.jsx [hunk 6]
```diff
@@ -82,18 +70,26 @@ const Footer = ({
...
             <div className="copyright">
-              ©{' '}
+              ©
+              {' '}
               <Link to="https://bluenove.com/offres/assembl/" target="_blank">
                 Assembl
-              </Link>{' '}
-              powered by{' '}
+              </Link>
+              {' '}
```
Comment (pascalhuynh) [inline assembl/static2/js/app/components/common/footer.jsx]: interpolation or I18n formatting would be appreciated

File: assembl/static2/js/app/components/common/postsAndContributorsCount.jsx [hunk 7]
```diff
@@ -20,7 +20,9 @@ type PostsAndContributorsCountProps = {
 
 export const Counter = ({ num, elemClassName, className }: CounterProps) => (
   <span className={elemClassName}>
-    {num} <span className={className} />
+    {num}
```
Comment (pascalhuynh) [inline assembl/static2/js/app/components/common/postsAndContributorsCount.jsx]: interpolation, I18n formatting or even CSS would be appreciated

File: assembl/static2/js/app/components/common/profileLine.jsx [hunk 8]
```diff
@@ -34,20 +34,19 @@ class ProfileLine extends React.PureComponent<Props> {
...
-              </div>
-            )}
+          {creationDate && locale && (
+            <div className="date">
+              {moment(creationDate)
+                .locale(locale)
+                .fromNow()}
+              {modified ? (
+                <span>
+                  {' - '}
```
Comment (pascalhuynh) [inline assembl/static2/js/app/components/common/profileLine.jsx]: interpolation, I18n formatting or even CSS would be appreciated

File: assembl/static2/js/app/components/administration/discussion/preferences/index.jsx (hunk 3 above)
Comment (felixsaintgenis) [reply assembl/static2/js/app/components/administration/discussion/preferences/index.jsx]: It's the same path without the unnecessary ../component 
It's a new rule that reformat the path to the easiest path

**Comment by pascalhuynh** [comment]
@felixsaintgenis should this PR be cancelled and closed ?

**Comment by felixsaintgenis** [comment]
Rest in peace little PR. You'll always be in our hearts

**Pull request closed without merging**